lale/lib/rasl/aggregate.py

```diff
@@ -32,9 +34,10 @@
 
 
 class _AggregateImpl:
-    def __init__(self, columns, group_by=[]):
+    def __init__(self, columns, group_by=[], exclude_value=None):
```

Comment:
Filter would work only if we use `Filter >> Aggregate` per column. Given that Aggregate is working column-wise for pandas, it might be similar in performance. For Spark, I will check if a combined aggregate speeds things up.

Reply:
Oh I see, I didn't think of the per-column complication. I should have known you had a good reason to make it a hyperparameter of Aggregate.

Reply:
Hm, I wonder if it would make sense to do a single Filter that removes an entire row with all of its columns if at least one of its columns is the exclude_value?

Reply:
I had tried that, but that doesn't match scikit-learn's output.